src/lib.rs

```diff
@@ -230,7 +279,7 @@ fn command<'a>(i: &'a [u8]) -> IResult<&'a [u8], Command<'a>> {
     // s [addr]
     // S sig[;addr]
     // t addr:PP,MM
-    // T thread-id
+    | parse_ping_thread => { |thread_id| Command::PingThread(thread_id) }
```

Comment:
It is a little weird that we are calling this parameter `thread_id`, but it is actually of type `ProcessId`.

Reply:
I changed the type to `ThreadId`, because that is how the RSP docs refer to these.